### PR TITLE
Adding code tab for Password input component

### LINKED ISSUE #1036: Add Code tab to Password input page

### PR DESCRIPTION
**Pull request by jsuarezgonz**
Closes #1036

### CODE REVIEW

File: website/pages/components/password-input/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Head from "next/head";
 import type { ReactElement } from "react";
-import PasswordInputUsagePage from "../../../screens/components/password-input/usage/PasswordInputUsagePage";
+import PasswordInputCodePage from "../../../screens/components/password-input/code/PasswordInputCodePage";
 import PasswordInputPageLayout from "../../../screens/components/password-input/PasswordInputPageLayout";
 
 const Usage = () => {
```
Comment (Lisa159) [inline website/pages/components/password-input/index.tsx:6]: Change variable name Usage to Index.

**Comment by Lisa159** [comment]
What do you think about adding a basic usage example?

File: website/screens/components/password-input/code/examples/errorHandling.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    setErrorMessage(error == undefined ? "" : "Error onChange");
+  };
+
+  const onBlur = ({ value, error }) => {
+    setValue(value);
+    setErrorMessage(error == undefined ? "" : "Error onBlur");
+  };
+
+  return (
+    <DxcInset space="small">
```
Comment (Lisa159) [inline website/screens/components/password-input/code/examples/errorHandling.ts:18]: Space prop should be "large".

Comment (jsuarezgonz) [reply website/screens/components/password-input/code/examples/errorHandling.ts]: Done

**Comment by jsuarezgonz** [comment]
>What do you think about adding a basic usage example?

I think the Controlled example is enough.

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**